exercises/concept/logs-logs-logs/.docs/introduction.md

```diff
@@ -1,6 +1,6 @@
 # Introduction
 
-The `rune` type in Go is an alias for `int32`. Given this underlying `int32` type, the `rune` type holds a signed 32-bit integer value. However, unlike an `int32` type, the integer value stored in a `rune` type is represents a single Unicode character.
+The `rune` type in Go is an alias for `int32`. Given this underlying `int32` type, the `rune` type holds a signed 32-bit integer value. However, unlike an `int32` type, the integer value stored in a `rune` type represents a single Unicode character.
```

Comment:
Could you please apply the fixes you did for this introduction file also for the `concepts/runes/introduction.md` and `concepts/runes/about.md` files? They contain largely the same content so I suspect they have the same typos.

Reply:
@junedev Of course! :) Just pushed the update. 👍